CONTRIBUTING.md

```diff
@@ -51,39 +51,59 @@ feat: add new feature
 
 ## Setup
 
+We use Node `18.18.0` and pnpm `8.15.4` to run the project.
+
 ```bash
 git clone https://github.com/dokploy/dokploy.git
 cd dokploy
-npm install
+pnpm install
 cp .env.example .env
 ```
 
 ## Development
 
 Is required to have **Docker** installed on your machine.
 
+
+### Setup
+
+We need to create the file system of dokploy and give the permissions.
+
 ```bash
-npm run dev
+sudo mkdir -p /etc/dokploy && sudo chmod 777 /etc/dokploy
```

Comment:
maybe we for development we can store those config files in repo directory (like `./.dokploy`, not in /etc? just for safety reasons

and instead of 777 i recommend using 700, cause making directory public to all users isn't necessary in this case

Reply:
Good sugestion!

Even if we remove the /etc/dokploy and move to the current dir where dokploy is located we dont need to give permissions!